Carthage.md

```diff
@@ -79,6 +80,9 @@ The frameworks contain static libraries that are linked at build time.
 
 - Some additional libraries/frameworks may be needed depending on which Firebase frameworks you use, for example: libc++.tbd, sqlite3.tbd, StoreKit.framework, etc. For more information, [go here](https://github.com/firebase/firebase-ios-sdk/issues/9#issuecomment-387947163).
 
+- For Crashlytics:
+    - To automatically upload your app's symbols so your app's crashes are symbolicated, download [upload-symbols](https://github.com/firebase/firebase-ios-sdk/raw/master/Crashlytics/upload-symbols) and [run](https://github.com/firebase/firebase-ios-sdk/raw/master/Crashlytics/run). Then continue to follow the Crashlytics documentation to add a new run script phase in your XCode project.
```

Comment:
Add link for Crashlytics documentation. 

Also, even though the above line breaks the rule, we try to follow a 100-character line limit in md files so they are easily readable in multiple editors.

Reply:
Update the lines so they do not exceed the 100-character line limit except for lines that end in with a hyperlink. 

I will keep this comment open as the new SDK documentation has not been published yet.

Reply:
Added link to https://firebase.google.com/docs/crashlytics/get-started-new-sdk?platform=ios